src/components/Editor/TableOfContents.vue

```diff
@@ -40,12 +40,7 @@ export default {
 	},
 	methods: {
 		goto(heading) {
-			this.$editor
-				.chain()
-				.focus()
-				.setTextSelection(heading.position)
-				.scrollIntoView()
-				.run()
+			document.getElementById(heading.id).scrollIntoView()
```

Comment:
This is slightly different from what we used to do. In particular we do not set the cursor anymore.

We could also use the  old way replacing `heading.position` with `heading.offset + 1`. I changed it as relying on the id seemed more robust to me than the offset at that time - but both should be fine now as we recompute and update the offset whenever the doc changes.